src/serialize.h

```diff
@@ -1142,6 +1146,24 @@ class ParamsStream
             return m_substream.template GetParams<P>();
         }
     }
+
+    //! Get reference to underlying stream.
+    auto& GetStream()
```

Comment:
Are you going to be using this method for other purposes besides tests and as an internal getter?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/28929#discussion_r1600617406

> Are you going to be using this method for other purposes besides tests and as an internal getter?

I could see some other uses. Initially this was just added for testing, but then marco pointed out it could be used as an internal getter in https://github.com/bitcoin/bitcoin/pull/28929#discussion_r1497229478 to avoid recursion in read, write, size, etc methods. The other case it might be useful is if you have have a custom stream type like a file or socket and want to call some method on that stream that ParamsStream() is not wrapping, without having to pass around separate pointers to both streams.